internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -354,6 +354,31 @@ type networkLoadBalancerConfig struct {
 	appDNSName           *string
 }
 
+func convertEnvSecurityGroupCfg(mft *manifest.Environment) template.SecurityGroupConfig {
+	securityGroupConfig, isSecurityConfigSet := mft.EnvSecurityGroup()
+	if isSecurityConfigSet {
```

Comment:
nit: we can flip the conditional to reduce nesting
```suggestion
	if !isSecurityConfigSet {
		return template.SecurityGroupConfig{}
	}
	ingress := make(// remaining code...)
```